src/index.ts

```diff
@@ -137,6 +146,38 @@ export class WaxJS {
     return this.user.account;
   }
 
+  private async getChainInfoByChainName(chainName: string): Promise<any> {
```

Comment:
let's add a public `getAvailableChains()` helper to make it easy to list the chains that can be switched to

Reply:
updated 👍